boot/freeldr/freeldr/lib/mm/meminit.c

```diff
@@ -243,7 +243,7 @@ MmCheckFreeldrImageFile(VOID)
     PIMAGE_OPTIONAL_HEADER OptionalHeader;
 
     /* Get the NT headers */
-    NtHeaders = RtlImageNtHeader(&__ImageBase);
+    NtHeaders = RtlpImageNtHeader(&__ImageBase);
```

Comment:
I know at least freeldr also has a `libsupp.c`, so you could also add the Rtl- versions there, and therefore avoid changing callers. That might also work for bootmgfw/rosload?
(you also wouldn't need `RtlpImageNtHeader` at all in that case)

Reply:
This sounds like a cleaner solution.